src/components/ebay-snackbar-dialog/index.marko

```diff
@@ -0,0 +1,18 @@
+import processHtmlAttributes from "../../common/html-attributes"
+
+<ebay-dialog-base
+    ...input
+    isModal=false
+    classPrefix="snackbar-dialog"
+    open=state.open
```

Comment:
Do `input.open`
Or you don't need it because you should be passing it through from `...input`